pkg/lint/rules/testdata/goodchart/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: badchart
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
+icon: "http://example.com/icon.png"
+annotations:
+  "hypper.cattle.io/release-name": "release"
+  "hypper.cattle.io/namespace": "namespace"
+  "hypper.cattle.io/shared-dependencies": |
+    - key: value
```

Comment:
This shouldn't pass the test, it should be `- name: value`.

Reply:
mmh, isnt this just checking if the `hypper.cattle.io/shared-dependencies` contains a valid yaml? 

The test should probably be expanded to not only validate that yaml but also for each entry in there try to load a `Dependency` object with those values.

I kind of wanted to make this simple, expanding it over time as its not an urgent matter and Im not sure if the shared stuff is final (like it could change format?? I dont know sorry)

Reply:
oh wait, is this because I messed up the Unmarshall in line 77??? WOW!Good catch, I almost messed up!